### PR TITLE
Transfered js codes in pixelorigin.html into a seperate

### PR DESCRIPTION
**Pull request by Jennifer-tech**
Fixes #0000 (<=== Add issue number here)#1215

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by 7malikk** [comment]
Hello @Jennifer-tech, kindly attach the number of the issue this PR solves, to give the reviewers ease. Thank you

**Comment by Jennifer-tech** [comment]
Ok, thank you

**Comment by Jennifer-tech** [comment]
But the issue number is already there

**Comment by Jennifer-tech** [comment]
Hi @jywarren @TildaDares Please kindly review my pull request #1216

File: README.md [hunk 1]
```diff
@@ -860,6 +860,9 @@ imgGroup = L.distortableCollection({
 ```
 
 ---
+## MISCELLENEOUS
```
Comment (jywarren) [inline README.md:863]: Hi @Jennifer-tech thank you for this PR! I believe this file is a separate issue. Could you reset it so it's not included in the PR? You could try this -- https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file or just copy the version from the publiclab main branch (https://github.com/publiclab/Leaflet.DistortableImage) over your own and add it so there are no changes shown here. Let me know if you need help!

Comment (Jennifer-tech) [reply README.md:863]: Going through it now, thank you for the feedback @jywarren

Comment (Jennifer-tech) [reply README.md]: Hi @jywarren, I have resolved the merge conflict. Please you can now review it

**Review by jywarren — APPROVED** [review]
Nicely done, thank you!

**Pull request merged**